ui/admin/src/app/libraries/libraries.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { HttpClientInMemoryWebApiModule } from 'angular-in-memory-web-api';
+
+import { TabsModule } from 'ngx-bootstrap/tabs';
+import { UiSwitchModule } from 'ngx-toggle-switch';
+
+import { InMemoryLibrariesDataService }  from './in-memory-libraries-data.service';
+import { LibrariesRoutingModule } from './libraries-routing.module';
+import { LibraryComponent } from './library/library.component';
+import { LibraryFormService } from './library-form.service';
+import { LibrariesService } from './libraries.service';
+import { MainComponent } from './main/main.component';
+
+import { environment } from '../../environments/environment';
+
+@NgModule({
+  providers: [
+    LibrariesService,
+    LibraryFormService
+  ],
+  declarations: [LibraryComponent, MainComponent],
+  imports: [
+    CommonModule,
+    LibrariesRoutingModule,
+    HttpClientModule,
+    environment.production ? [] : HttpClientInMemoryWebApiModule.forRoot(
+      InMemoryLibrariesDataService, { dataEncapsulation: false, put204: false }
```

Comment:
it would be great to have some comments here to explain the options

Reply:
Documentation added.